Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -82,12 +84,70 @@ def tap_new
 
             - run: brew test-bot --only-formulae
               if: github.event_name == 'pull_request'
+
+            - name: Upload bottles as artifact
+              if: always() && github.event_name == 'pull_request'
+              uses: actions/upload-artifact@main
+              with:
+                name: bottles
+                path: '*.bottle.*'
+    YAML
+
+    actions_publish = <<~YAML
+      name: brew pr-pull
+      on:
+        pull_request_target:
+          types:
+            - labeled
+      jobs:
+        pr-pull:
+          if: contains(github.event.pull_request.labels.*.name, 'ok')
```

Comment:
```suggestion
          if: contains(github.event.pull_request.labels.*.name, 'homebrew-publish-bottles')
```

or something? I'm pretty open to anything but something more descriptive and/or homebrew-specific might be nice.

Reply:
Imma name it as `pr-pull` by default, it matches the workflow name and is short enough.